src/static/riot/profiles/user_detail.tag

```diff
@@ -0,0 +1,177 @@
+                             }
+                         ]
+                     },
+                     twitter_url: {
+                         identifier: 'twitter_url',
+                         optional: true,
+                         rules: [
+                             {
+                                 type: 'url',
+                                 prompt: 'Please enter a valid twitter url. Example: https://twitter.com/BobRoss'
+                             }
+                         ]
+                     },
+                     linkedin_url: {
+                         identifier: 'linkedin_url',
+                         optional: true,
+                         rules: [
+                             {
+                                 type: 'url',
+                                 prompt: 'Please enter a valid url. Example: https://www.linkedin.com/in/john-doe'
+                             }
+                         ]
+                     },
+                     github_url: {
+                         identifier: 'github_url',
+                         optional: true,
+                         rules: [
+                             {
+                                 type: 'url',
+                                 prompt: 'Please enter a valid url. Example: https://github.com/john-doe'
+                             }
+                         ]
+                     },
+                 },
+                 onSuccess: function (){
+                     _.extend(self.selected_user, $('#user-form').form('get values'))
+                     CODALAB.api.update_user_details(self.selected_user.id, self.selected_user)
+                         .done( data => {
+                             toastr.success("Details Saved")
+                             window.location.href = data
+                         })
+                         .fail(errors =>{
+                             toastr.error(JSON.stringify(errors))
+                             self.refs.submit_button.disabled = false
+                         })
+                     return false
+                 }
+             })
+            self.photo_name = typeof self.photo == 'undefined' || self.photo === null ? null : self.photo.replace(/\\/g, '/').replace(/.*\//, '')
+            // Draw in logo filename as it's changed
+            $(self.refs.photo).change(function () {
+                self.logo_file_name = self.refs.photo.value.replace(/\\/g, '/').replace(/.*\//, '')
+                self.update()
+                getBase64(this.files[0]).then(function (data) {
+                    self.selected_user['photo'] = JSON.stringify({file_name: self.logo_file_name, data: data})
+                })
+            })
+            self.update()
+        })
+
+        self.save = () => {
+            self.refs.submit_button.disabled = true
+            $('#user-form').form('validate form')
+        }
+    </script>
+</user-detail>
```

Comment:
Submit button becomes disabled if the form is invalid and never becomes re-enabled.


To reproduce:
### 1. Type invalid url into linkedIn field and then click submit. Submit button is disabled as it should be:
![image](https://user-images.githubusercontent.com/21694127/99607593-f07b3c80-29c0-11eb-8bba-a2696fd75268.png)

### 2. Type valid url into linkedIn field. Error state disappears for field, but no re-enable of the submit button:
![image](https://user-images.githubusercontent.com/21694127/99607728-37693200-29c1-11eb-8243-4336770acb8c.png)

Reply:
Fixed